Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:8
```

Comment:
any reason not `node:8-alpine` here? (just in the interests of smaller image size)

Reply:
Now done, though was more challenging than expected...